src/index.ts

```diff
@@ -256,8 +257,17 @@ export class Project {
     this.pkg.version = value;
   }
 
-  async write() {
+  mergeFiles(dirJSON: fixturify.DirJSON) {
```

Comment:
I think we need to test this capability as well

Reply:
Yep I debated that. It's pretty simple, and is essentially tested via the `write(dirJSON)` test I added. I'm happy to have a separate test for it though.

Reply:
@scalvert, I think the only thing I would recommend capturing is the precedence during the merge between current files and the files provided via argument.

Reply:
Added a test to ensure the ordering.